File: Client/WebClient/src/app/models/dtos/port/otauAttachedDto.ts
import { ReturnCode } from "../../enums/returnCode";

export class OtauAttachedDto {
  otauId: string;

  rtuId: string;
  isAttached: boolean;

  returnCode: ReturnCode;
  errorMessage: string;
  serial: string;
  portCount: number;
}

File: Shared/Dto/R2D2C/OtauAttachedDto.cs
﻿using System;
using System.Runtime.Serialization;

namespace Iit.Fibertest.Dto
{
    [DataContract]
    public class OtauAttachedDto
    {
        [DataMember]
        public Guid OtauId { get; set; }

        [DataMember]
        public Guid RtuId { get; set; }

        [DataMember]
        public bool IsAttached { get; set; }

        [DataMember]
        public ReturnCode ReturnCode { get; set; }
        [DataMember]
        public string ErrorMessage { get; set; }

        [DataMember]
        public string Serial { get; set; }

        [DataMember]
        public int PortCount { get; set; }
    }
}